gamemode/load_modules.lua

```diff
@@ -2,23 +2,52 @@ local moduleDir = GM.FolderName .. "/gamemode/modules/"
 
 local _, modules = file.Find( moduleDir .. "*", "LUA" )
 
+print( "[RVR] Loading modules..." )
 for _, moduleName in pairs( modules ) do
     local path = moduleDir .. moduleName .. "/"
     local files, _ = file.Find( path .. "*.lua", "LUA" )
-    
-    for _, file in pairs( files ) do
-        local fullPath = path .. file
+    local isServer = false
+    local isClient = false
+
+    for _, fileName in pairs( files ) do
+        local fullPath = path .. fileName
         if CLIENT then
             include( fullPath )
-        elseif string.StartWith( file, "sh_" ) then
+        elseif string.StartWith( fileName, "sh_" ) then
+            isClient = true
+            isServer = true
+
             include( fullPath )
             AddCSLuaFile( fullPath )
-        elseif string.StartWith( file, "sv_" ) then
+        elseif string.StartWith( fileName, "sv_" ) then
+            isServer = true
+
             include( fullPath )
-        elseif string.StartWith( file, "cl_" ) then
+        elseif string.StartWith( fileName, "cl_" ) then
+            isClient = true
+
             AddCSLuaFile( fullPath )
         end
     end
+
+    if SERVER then
+        local realmStr
+        if isServer and isClient then
+            realmStr = "shared"
+        elseif isServer then
+            realmStr = "server"
+        elseif isClient then
+            realmStr = "client"
+        else
+            print( "[RVR] Warning, empty module: " .. moduleName )
+            continue
```

Comment:
*Jenkins is typing...*

Reply:
Do we actually not like continue for any reason other than jenkins being bad?
continue is genuinely really useful, helps support a RERO style of programming in loops
A *CECO* of sorts

Reply:
I like it but Jenkins will not, I personally prefer the use of continue since it leaves the code looking generally more clean but alas this is on the CFC repository so we at least have to follow its metrics.

Reply:
Let's just keep it in then, it's far better than more confusing logic...

Reply:
We could even look into getting jenkins fixed for this repo

Reply:
its  not a problem with jenkins being old, its a problem with continue not being in normal lua and lua linters dont like that
i think continue could be removed here without too much impact

Reply:
I think its strange to say that. I mean it's not like we're going to be using these scripts ever outside of Garry's Mod...

Reply:
its not strange at all, im explaining why jenkins complains about continue

Reply:
We don't have string indexing in normal lua as well, but we still use that because it makes the code nicer. I don't see why this is different, especially since we don't have jenkins on this project.

Reply:
I just really don't like making our code worse because jenkins isn't set up to glua correctly.

Reply:
yeah but lua linters usually dont have a problem with string indexing, i usually avoid it because of that. doesnt really matter though, do whatever

Reply:
The non-continue alternative is as follow:
```lua
        local realmStr
        if isServer and isClient then
            realmStr = "shared"
        elseif isServer then
            realmStr = "server"
        elseif isClient then
            realmStr = "client"
        else
            print( "[RVR] Warning, empty module: " .. moduleName )
        end

        if realmStr then
            print( "[RVR] Loaded " .. realmStr .. " module: " .. moduleName )
        end
```

Reply:
thats not really that bad

Reply:
In this case, thats not actually too bad, but there are definately other cases where `continue` is much nicer, and this conversation kinda goes for all of those as well.

Reply:
I remove it here, but I don't think we should remove the possibility of using `continue` in this project.

*This battle isn't over*